### ISSUE TITLE
Unbale to import the saml remote SP metadata XML to the AM for particular cot

### ISSUE THREAD

**Issue by Biswajit0180** [rockcarver/frodo-cli#401]
## Frodo CLI version
Provide output of `frodo -v`

 frodo -v
You are running the binary release.
Installed versions:
cli: v1.0.1
lib: v1.1.0
node: v18.5.0

## Describe the issue
A clear and concise description of what the issue is.

Unbale to import the saml remote SP metadata XML to the AM for particular cot

frodo saml import https://Host/am example amadmin '***********' --entity-id https://example123.com -f exportmetadata_exp123.xml --raw -m classic  -k

Importing raw provider "https://example123.com" into realm "example"...
Importing SAML Entity exportmetadata_exp123.xml...

But actually its not uploading to AM.

**Comment by vscheuber** [comment]
@Biswajit0180 please let me know if we can close this issue or if you think this is something that needs to be fixed still.

**Comment by hfranklin** [comment]
@Biswajit0180 Is this still an issue?

**Comment by Biswajit0180** [comment]
> [@Biswajit0180](https://github.com/Biswajit0180) Is this still an issue?

Yes same issue is present on new version as well. I means Remote SP always send the SP metadata on XML file. so using this saml import doesn't work. Its expecting json file.
